### ISSUE TITLE
FW: Digital UNIX problem with Perl 5.8.0

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#6239]
Migrated from [rt.perl.org#20487](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=20487) (status was 'rejected')

Searchable as RT20487$

**Comment by p5pRT** [comment]
### From Wieslaw.Broszczak@itu.int



\-\-\-\-\-Original Message\-\-\-\-\-
From&#8203;: Attila Bruncsák \[mailto&#8203;:attila@&#8203;coral\.itu\.ch\]
Sent&#8203;: mercredi\, 22\. janvier 2003 14&#8203;:54
To&#8203;: broszczak@&#8203;itu\.int
Subject&#8203;: Digital UNIX problem with Perl 5\.8\.0


There was an error in "make test" of the latest stable version of the Perl\.
I got reports about the function strxfrm\(\)\.
I was taking care to remove all the locale related variables and
it did not help\. I was also seting LC\_ALL=C with no more success\.
Could you help please?
When I run \./perl harness I got the following error&#8203;:

\.\./lib/locale\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.ok 98/117strxfrm\(\) gets absurd at
\.\./lib/locale\.t line 332\, \<LOCALES> line 22\.
\.\./lib/locale\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.\.dubious

&nbsp;        Test returned status 255 \(wstat 65280\, 0xff00\)
DIED\. FAILED tests 99\-117
&nbsp;        Failed 19/117 tests\, 83\.76% okay


Failed Test     Stat Wstat Total Fail  Failed  List of Failed


---


---
\.\./lib/locale\.t  255 65280   117   19  16\.24%  99\-117
41 tests and 271 subtests skipped\.
Failed 1/712 test scripts\, 99\.86% okay\. 19/68665 subtests failed\, 99\.97%
okay\.


And here is the content of the myconfig script&#8203;:


\#\!/bin/sh

\# This script is designed to provide a handy summary of the configuration
\# information being used to build perl\. This is especially useful if you
\# are requesting help from comp\.lang\.perl\.misc on usenet or via mail\.

\# Note that the text lines /^Summary of/ \.\. /^\\s\*$/ are copied into
Config\.pm\.
cat \<\<'\!NO\!SUBS\!'
Summary of my perl5 \(revision 5\.0 version 8 subversion 0\) configuration&#8203;:
&nbsp;  Platform&#8203;:
&nbsp;    osname=dec\_osf\, osvers=4\.0f\, archname=alpha\-dec\_osf
&nbsp;    uname='osf1 coral\.itu\.ch v4\.0 1229 alpha '
&nbsp;    config\_args='\-des \-Dcc=cc \-Dinstallprefix=/usr/site'
&nbsp;    hint=recommended\, useposix=true\, d\_sigaction=define
&nbsp;    usethreads=undef use5005threads=undef useithreads=undef
usemultiplicity=undef
&nbsp;    useperlio=define d\_sfio=undef uselargefiles=define usesocks=undef
&nbsp;    use64bitint=define use64bitall=define uselongdouble=undef
&nbsp;    usemymalloc=n\, bincompat5005=undef
&nbsp;  Compiler&#8203;:
&nbsp;    cc='cc'\, ccflags ='\-std \-D\_INTRINSICS \-fprm d \-ieee \-I/usr/local/include
\-DLANGUAGE\_C'\,
&nbsp;    optimize='\-O4'\,
&nbsp;    cppflags='\-std \-D\_INTRINSICS \-ieee \-I/usr/local/include \-DLANGUAGE\_C'
&nbsp;    ccversion='V5\.9\-011'\, gccversion=''\, gccosandvers=''
&nbsp;    intsize=4\, longsize=8\, ptrsize=8\, doublesize=8\, byteorder=12345678
&nbsp;    d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=8
&nbsp;    ivtype='long'\, ivsize=8\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\,
lseeksize=8
&nbsp;    alignbytes=8\, prototype=define
&nbsp;  Linker and Libraries&#8203;:
&nbsp;    ld='ld'\, ldflags =' \-L/usr/local/lib'
&nbsp;    libpth=/usr/local/lib /usr/shlib /usr/ccs/lib /usr/lib/cmplrs/cc
/usr/lib /var/shlib
&nbsp;    libs=\-ldbm \-ldb \-lm \-lutil
&nbsp;    perllibs=\-lm \-lutil
&nbsp;    libc=/usr/shlib/libc\.so\, so=so\, useshrplib=true\, libperl=libperl\.so
&nbsp;    gnulibc\_version=''
&nbsp;  Dynamic Linking&#8203;:
&nbsp;    dlsrc=dl\_dlopen\.xs\, dlext=so\, d\_dlsymun=undef\, ccdlflags='
\-Wl\,\-rpath\,/usr/local/lib/perl5/5\.8\.0/alpha\-dec\_osf/CORE'
&nbsp;    cccdlflags=' '\, lddlflags='\-shared \-expect\_unresolved "\*" \-O4 \-msym \-std
\-s \-L/usr/local/lib'

\!NO\!SUBS\!

**Comment by p5pRT** [comment]
### From spider@Orb.Nashua.NH.US

\[Problems with locales on Tru64 UNIX V4\.0F\.\]

Some of the system locales \(especially with I18N 'world\-wide' subsets loaded\)
will cause this\.  On my V5\.1A system\, the following ones did&#8203;:
&nbsp;    ca\_ES\.UTF\-8
&nbsp;    he\_IL\.ISO8859\-8@&#8203;ucs4
&nbsp;    he\_IL@&#8203;ucs4
&nbsp;    iw\_IL\.ISO8859\-8@&#8203;ucs4
&nbsp;    iw\_IL@&#8203;ucs4

Basically\, if that's the only test in 'make test' which is failing\, your
perl is working just fine\.  Don't worry about it\.

\-\- 
Spider Boardman \(at home\)              spider@&#8203;Orb\.Nashua\.NH\.US
The management \(my cats\) made me say this\.    http&#8203;://users\.rcn\.com/spiderb/
PGP public key fingerprint&#8203;: 96 72 D2 C6 E0 92 32 89  F6 B2 C2 A0 1C AB 1F DC

**Comment by p5pRT** [comment]
### From @jkeenan

On Wed Jan 22 10&#8203;:05&#8203;:02 2003\, spider@&#8203;Orb\.Nashua\.NH\.US wrote&#8203;:
> \[Problems with locales on Tru64 UNIX V4\.0F\.\]
> 
> Some of the system locales \(especially with I18N 'world\-wide' subsets
> loaded\)
> will cause this\.  On my V5\.1A system\, the following ones did&#8203;:
>     ca\_ES\.UTF\-8
>     he\_IL\.ISO8859\-8@&#8203;ucs4
>     he\_IL@&#8203;ucs4
>     iw\_IL\.ISO8859\-8@&#8203;ucs4
>     iw\_IL@&#8203;ucs4
> 
> Basically\, if that's the only test in 'make test' which is failing\,
> your
> perl is working just fine\.  Don't worry about it\.
> 

This response appears reasonable\, and there has been no further
discussion in this ticket in nine years\.

Can we close it?

Thank you very much\.
Jim Keenan

**Comment by p5pRT** [comment]
### From @khwilliamson

On 04/21/2012 10&#8203;:28 AM\, James E Keenan via RT wrote&#8203;:
> On Wed Jan 22 10&#8203;:05&#8203;:02 2003\, spider@&#8203;Orb\.Nashua\.NH\.US wrote&#8203;:
>> \[Problems with locales on Tru64 UNIX V4\.0F\.\]
>>
>> Some of the system locales \(especially with I18N 'world\-wide' subsets
>> loaded\)
>> will cause this\.  On my V5\.1A system\, the following ones did&#8203;:
>>     ca\_ES\.UTF\-8
>>     he\_IL\.ISO8859\-8@&#8203;ucs4
>>     he\_IL@&#8203;ucs4
>>     iw\_IL\.ISO8859\-8@&#8203;ucs4
>>     iw\_IL@&#8203;ucs4
>>
>> Basically\, if that's the only test in 'make test' which is failing\,
>> your
>> perl is working just fine\.  Don't worry about it\.
>>
>
> This response appears reasonable\, and there has been no further
> discussion in this ticket in nine years\.
>
> Can we close it?

I vote to close it
>
> Thank you very much\.
> Jim Keenan
>
>
>
>
> \-\-\-
> via perlbug&#8203;:  queue&#8203;: perl5 status&#8203;: open
> https://rt-archive.perl.org/perl5/Ticket/Display.html?id=20487
>

**Comment by p5pRT** [comment]
### From @jkeenan

On Sat Apr 21 11&#8203;:43&#8203;:37 2012\, public@&#8203;khwilliamson\.com wrote&#8203;:
> On 04/21/2012 10&#8203;:28 AM\, James E Keenan via RT wrote&#8203;:
> > On Wed Jan 22 10&#8203;:05&#8203;:02 2003\, spider@&#8203;Orb\.Nashua\.NH\.US wrote&#8203;:
> >> Basically\, if that's the only test in 'make test' which is failing\,
> >> your
> >> perl is working just fine\.  Don't worry about it\.
> >>
> >
> > This response appears reasonable\, and there has been no further
> > discussion in this ticket in nine years\.
> >
> > Can we close it?
> 
> I vote to close it

Thanks\. Done\.

**Comment by p5pRT** [comment]
@jkeenan - Status changed from 'open' to 'rejected'
